### PR TITLE
Expose dimensions in pc1.19 codec

### PR DESCRIPTION
**Pull request by extremeheat**

### CODE REVIEW

File: lib/pc/index.js [hunk 1]
```diff
@@ -29,6 +29,22 @@ module.exports = (data, staticData) => {
...
+        data.dimensionsById = {}
+        data.dimensionsByName = {}
+        for (const { name, id, element } of dimensions) {
+          const n = name.replace('minecraft:', '')
+          const d = {
+            name: n,
+            minY: element.min_y,
+            height: element.height
+          }
+          data.dimensionsById[id] = d
```
Comment (rom1504) [inline lib/pc/index.js:43]: What will be values of this not in 1.19 ?

Is it possible to have default values for other versions?

Comment (extremeheat) [reply lib/pc/index.js:43]: There is no dimensions data for versions below 1.19 but static data for older versions could be added to Minecraft-data I think.

Comment (rom1504) [reply lib/pc/index.js:43]: For now let's either:
* Hardcore some values for all versions that don't have dimensions data
* Or document this is only supported for 1.19 for now

**Comment by extremeheat** [comment]
documented that it's 1.19+

**Pull request merged**